src/renderer/components/basic.lua

```diff
@@ -51,7 +51,14 @@ function basicTextComponent:render(surf, position, styles, resolver)
   end
 
   local cY = position.top + topPad
-
+  
+  local totalSize = 1;
+  local parent = self;
+  while (parent ~= null) do
```

Comment:
It's `nil` not `null`

Reply:
Also just say, `while parent do`, it's cleaner that way

Reply:
Sorry, It's from my java programming